overlay/k8s/cncf/etc/k8sconfig/calico.yaml

```diff
@@ -0,0 +1,4648 @@
+---
+# Source: calico/templates/calico-config.yaml
+# This ConfigMap is used to configure a self-hosted Calico installation.
+kind: ConfigMap
```

Comment:
this doesn't belong here - in the core there shouldn't be any provider specific configuration

Reply:
Ok, so better to move provider overlay to provider repo ? I will fix this

Reply:
yes exactly, this is provider specific